### ISSUE TITLE
Issue with enabling security web service

### ISSUE THREAD

**Issue by xDarkHawkx** [AdguardTeam/AdGuardHome#6831]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to report a bug and not [ask a question or ask for help](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a)

- [X] I have set up AdGuard Home correctly and [configured clients to use it](https://github.com/AdguardTeam/AdGuardHome/wiki/Clients). (Use the [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions/categories/q-a) for help with installing and configuring clients.)

### Platform (OS and CPU architecture)

Windows, AMD64 (aka x86_64)

### Installation

GitHub releases or script from README

### Setup

On a router, DHCP is handled by the router

### AdGuard Home version

v0.108.0-b.53

### Action

Turning on "use AdGuard browsing security web service" breaks AdGuard home and DNS stops routing turning it back off everything works as normal. 

### Expected result

DNS to route

### Actual result

It doesn't 

### Additional information and/or screenshots

![image](https://github.com/AdguardTeam/AdGuardHome/assets/163894970/4012eb02-994c-48da-ab3b-abac8d1176be)

**Comment by xDarkHawkx** [comment]
I have it running on a Lenovo mini computer running Windows 11. 
Adguard home is running as a service and works fine until I turn on "security web service" then DNS stops routing till I either turn that setting off or disable Adgaurd in general.

**Comment by xDarkHawkx** [comment]
AdGuard support asked me to reach out here. If I need to share more information let me know. Thanks everyone!

**Comment by xDarkHawkx** [comment]
This seems to be fixed after I just updated to Version: v0.108.0-b.54 that was just released 4 hours ago. I'll keep monitoring it and update everyone if it becomes an issue again. Thanks!

**Comment by Cebeerre** [comment]
Ok, closing the issue then. Thanks !

**Comment by xDarkHawkx** [comment]
> Ok, closing the issue then. Thanks !

I'm having the issue again. I even tried reinstalling Adguardhome on a new device and other versions. It quits working only when I enable security web service. 

Thanks!

**Comment by xDarkHawkx** [comment]
I have it running as a service on a Windows 11 box. Version:
![image](https://github.com/AdguardTeam/AdGuardHome/assets/163894970/dfe1d5f1-a237-4f40-8054-a11047260604)

Network settings of the host:
![image](https://github.com/AdguardTeam/AdGuardHome/assets/163894970/18c6bfd9-51e3-426a-9557-45cfb3338d00)

Network settings of my Unifi firewall:
![image](https://github.com/AdguardTeam/AdGuardHome/assets/163894970/517e41f3-11c0-47e9-87d9-a63829e57023)

And my config (the .yaml file)
![image](https://github.com/AdguardTeam/AdGuardHome/assets/163894970/4dc07eab-83a5-44f0-bf21-c2246e98dcc8)


If I need to take any logs or share anything else to diagnose this let me know!

**Comment by xDarkHawkx** [comment]
It also looks like others have reported the same issue:
https://github.com/AdguardTeam/AdGuardHome/issues/6817

**Comment by xDarkHawkx** [comment]
And another https://github.com/AdguardTeam/AdGuardHome/issues/5213#issuecomment-1331193106

**Comment by xDarkHawkx** [comment]
So I may have figured it out based on this comment: https://github.com/AdguardTeam/AdGuardHome/issues/3029#issuecomment-829101290 I couldn't access that page so I tried turning random security services off on my Unifi firewall. I turned off "country restrictions" and it seemingly started working again as well did turning on  security web service started working again. I'll continue to monitor. 

I did a reverse DNS lookup and it looks like https://dns.adguard.com/ is based out of Cypres. I honestly don't remember if I had that on my block list. But I'll continue testing. 

![image](https://github.com/AdguardTeam/AdGuardHome/assets/163894970/38fdd71a-3086-4005-87fa-e5c617a2470c)


Thanks!
